sim/druid/druid.go

```diff
@@ -121,13 +130,17 @@ type Druid struct {
 	SolarEclipseProcAura     *core.Aura
 	LunarEclipseProcAura     *core.Aura
 	WildStrikesBuffAura      *core.Aura
+	Tank2PieceAqAura         *core.Aura
+	Tank2PieceAqProcAura     *core.Aura
```

Comment:
Are these two auras used outside of the set bonus code? If not you should be able to remove these and just store them as locals